src/logcollector/read_nmapg.c

```diff
@@ -127,22 +127,23 @@ void *read_nmapg(logreader *lf, int *rc, int drop_it) {
     int final_msg_s;
     int need_clear = 0;
 
-    char str[OS_MAXSTR + 1];
-    char final_msg[OS_MAXSTR + 1];
-    char buffer[OS_MAXSTR + 1];
+    char str[OS_MAXSTR - OS_LOG_HEADER];
```

Comment:
Why the `- OS_LOG_HEADER`? Can't we use an intermediate macro with a descriptive name instead?

Reply:
Got it. In that case I think we should define a new macro for this value (`OS_MAXSTR - OS_LOG_HEADER`) something like
```
#define OS_MAX_LOG (OS_MAXSTR - OS_LOG_HEADER)
```
What do you think?